python/rikai/mixin.py

```diff
@@ -90,6 +90,51 @@ def __matmul__(self, style: Union[dict, "rikai.viz.Style"]) -> Drawable:
         return style(self)
 
 
+class ComposableDrawable:
+    """Mixin for a drawable that can be composed with others"""
+
+    @abstractmethod
+    def xy_min(self):
```

Comment:
rename to left_top ? xy_min is not straghtforward.

Reply:
good idea

Reply:
it seems you changed two of them into left_top

Reply:
and there are still two xy_min to be replaced by left_top